admin/admin-service/src/test/java/io/mosip/admin/bulkdataupload/batch/CustomExcelRowMapperTest.java

```diff
@@ -0,0 +1,184 @@
+
+        CustomExcelRowMapper<Object> customExcelRowMapper = new CustomExcelRowMapper<>(conversionService,
+                new CustomValidatorBean());
+        customExcelRowMapper.setTargetType(Object.class);
+        RowSet rowSet = mock(RowSet.class);
+        when(rowSet.getProperties()).thenThrow(new ConstraintViolationException(new HashSet<>()));
+        customExcelRowMapper.mapRow(rowSet);
+        verify(rowSet).getProperties();
+    }
+
+
+    @Test
+    public void createBinder_withApplicationConversionService_returnSuccessResponse() {
+        ApplicationConversionService conversionService = new ApplicationConversionService();
+        DataBinder actualCreateBinderResult = (new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()))
+                .createBinder("Target");
+        assertFalse(actualCreateBinderResult.isIgnoreUnknownFields());
+        assertFalse(actualCreateBinderResult.isIgnoreInvalidFields());
+        assertTrue(actualCreateBinderResult.isAutoGrowNestedPaths());
+        assertNull(actualCreateBinderResult.getValidator());
+        assertEquals("Target", actualCreateBinderResult.getTarget());
+        assertEquals("target", actualCreateBinderResult.getObjectName());
+        assertTrue(actualCreateBinderResult.getBindingErrorProcessor() instanceof DefaultBindingErrorProcessor);
+        assertEquals(256, actualCreateBinderResult.getAutoGrowCollectionLimit());
+    }
+
+    @Test
+    public void createBinder_withDefaultFormattingConversionService_returnSuccessResponse() {
+        DefaultFormattingConversionService conversionService = new DefaultFormattingConversionService(
+                mock(StringValueResolver.class), true);
+
+        DataBinder actualCreateBinderResult = (new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()))
+                .createBinder("Target");
+        assertFalse(actualCreateBinderResult.isIgnoreUnknownFields());
+        assertFalse(actualCreateBinderResult.isIgnoreInvalidFields());
+        assertTrue(actualCreateBinderResult.isAutoGrowNestedPaths());
+        assertNull(actualCreateBinderResult.getValidator());
+        assertEquals("Target", actualCreateBinderResult.getTarget());
+        assertEquals("target", actualCreateBinderResult.getObjectName());
+        assertTrue(actualCreateBinderResult.getBindingErrorProcessor() instanceof DefaultBindingErrorProcessor);
+        assertEquals(256, actualCreateBinderResult.getAutoGrowCollectionLimit());
+    }
+
+    @Test
+    public void findPropertyName_withValidRequest_thenPass() {
+        ApplicationConversionService conversionService = new ApplicationConversionService();
+
+        Object bean = "Bean";
+        String key = "Key";
+
+        ReflectionTestUtils.invokeMethod(new CustomExcelRowMapper<>(conversionService, new CustomValidatorBean()),
+                "findPropertyName", bean, key);
+
+    }
+
+    @Test
+    public void findPropertyName_withNullRequest_returnNull() {
```

Comment:
Assertion is missing in this test case

Reply:
This test case is for a private method, Since direct access to private methods for assertions is not feasible, I've utilized ReflectionTestUtils to ensure the method is invoked correctly